test/extended/networking.sh

```diff
@@ -24,7 +24,6 @@ NETWORKING_E2E_MINIMAL="${NETWORKING_E2E_MINIMAL:-}"
 
 DEFAULT_SKIP_LIST=(
   # TODO(marun) This should work with docker >= 1.10
-  "openshift router"
   "\[Feature:Federation\]"
```

Comment:
@marun, does that comment apply to both router and federation tests, and if so, should we enable both of them now?